.cicd/docker/amazonlinux-2.dockerfile

```diff
@@ -10,6 +10,7 @@ RUN curl -LO https://cmake.org/files/v3.13/cmake-3.13.2.tar.gz && \
     make -j $(nproc) && \
     make install && \
     rm -f /cmake-3.13.2.tar.gz && rm -rf /cmake-3.13.2
+
```

Comment:
This is an example of an unnecessary change that makes peer review more difficult. Since we are making changes anyways and, since there aren't spaces between the other Dockerfile commands, let's delete these empty lines.